airflow-core/src/airflow/ui/public/i18n/locales/pl/dag.json

```
@@ -39,6 +39,11 @@
     "viewInExternal": "Zobacz logi w {{name}} (próba {{attempt}})",
     "warning": "WARNING"
   },
+  "navigation": {
+    "jump": "Strona: Shift+{{arrow}}",
```

```suggestion
    "jump": "Przeskocz: Shift+{{arrow}}",
```
OR
```suggestion
    "jump": "Przejdź: Shift+{{arrow}}",
```
? Not sure I know the context here, just looking at english files

---

I like Przeskocz. Was struggling about that one